tests/test/test_checkout_volunteer.py

```
@@ -0,0 +1,20 @@
+import pytest
+from pages.checkout_page import CheckOutPage
+from utilities.fixtures import login_with_volunteer
+from utilities.fixtures import driver
+
+
+@pytest.fixture(scope="function")
+def checkout_page(driver):
+    return CheckOutPage(driver)
+
+@pytest.mark.usefixtures('login_with_volunteer')
+def test_checkout(driver, checkout_page):
```

Ruff (0.12.2)

Redefinition of unused `driver` from line 4

(F811)